src/PKSim.Matlab/ApplicationStartup.cs

```diff
@@ -47,29 +47,27 @@ private void initializeForMatlab()
 
       private static void redirectNHibernateAssembly()
       {
-         redirectAssembly("NHibernate", new Version(4, 1, 0, 4000), "aa95f207798dfdb4");
+         redirectAssembly("NHibernate", new Version(5, 2, 0, 0), "aa95f207798dfdb4");
```

Comment:
what is this aa95f207798dfdb4 ?

Reply:
This is the nhiberbate package id (see app.config). This code ensures that the right dll is loaded. We had issues when loading from Matlab